.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -485,6 +516,130 @@ jobs:
           follow-up-finalization-check: true
         timeout-minutes: 15
 
+  run-e2e-version-upgrade-catchup-one-node-late:
+    needs: [build-test-docker, build-cliain-image]
+    name: Run a test where one of the nodes learns of version-upgrade during catchup
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Download artifact with docker image for aleph-node
+        uses: actions/download-artifact@v2
+        with:
+          name: aleph-test-docker
+
+      - name: Load node docker image
+        shell: bash
+        run: docker load -i aleph-node.tar
+
+      - name: Download artifact with docker image for cliain
+        uses: actions/download-artifact@v2
+        with:
+          name: cliain-docker
+
+      - name: Load cliain docker image
+        shell: bash
+        run: docker load -i cliain.tar
+
+      - name: Call catchup_test.sh
+        run: |
+          chmod +x ./scripts/catchup_version_upgrade_test.sh
```

Comment:
can't we just add this permission to the file in git and abandon this step in all 4 job definitions?

Reply:
`Dockerfile` for cliain handles it. I'll remove it.